src/Persistence/SiteLinkBasedStatementsLookup.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Persistence;
+
+use ProfessionalWiki\WikibaseFacetedSearch\Application\StatementsLookup;
+use Wikibase\DataModel\Entity\Item;
+use Wikibase\DataModel\Services\Lookup\EntityLookup;
+use Wikibase\DataModel\Statement\StatementList;
+use Wikibase\Lib\Store\SiteLinkLookup;
+use WikiPage;
+
+class SiteLinkBasedStatementsLookup implements StatementsLookup {
```

Comment:
The newer Wikibase code is using `Sitelink` instead of `SiteLink`, which I suppose is clearer than the original capitalization, making it obvious that it's about the single domain concept rather than "site" and "link"

Reply:
Mostly an FYI. We are still consistent with most WB code, so unclear if we should change here

Reply:
I was wondering about this. We're already using both capitalizations. Maybe we should just rename all our symbols to the lowercase `Sitelink`, and then only use uppercase `SiteLink` when calling or accessing symbols in Wikibase.

Reply:
Using `Sitelink` everywhere seems good